src/instrumenter/instrumentation_context.ts

```diff
@@ -244,4 +247,91 @@ export class InstrumentationContext {
             transCtx.finalize();
         }
     }
+
+    private customMapLibrary = new Map<
+        string,
+        [ContractDefinition, StructDefinition, Map<string, FunctionDefinition>, TypeNode, TypeNode]
+    >();
```

Comment:
Maybe it would be better to move all of the fields to the class header? It would be easier to spot all of them in one sequence (easy read-scanning).

Reply:
I think I fixed it in b4b6be9? Not sure. You check and resolve